sites/public/next.config.js

```diff
@@ -51,6 +52,7 @@ module.exports = withCSS(
             cacheRevalidate: process.env.CACHE_REVALIDATE
               ? Number(process.env.CACHE_REVALIDATE)
               : 60,
+            showAPPLinks: process.env.SHOW_APP_LINKS === "TRUE",
```

Comment:
nit: switch to `showAppLinks` so that it's true camelcase

Reply:
Resolved.